### ISSUE TITLE
Feature Request: STFT

### ISSUE THREAD

**Issue by dhgrs** [chainer/chainer#5036]
Backpropagatable STFT(short time fourier transform) is used in Parallel WaveNet[1] and other papers like [2]. Chainer has backpropagatable FFT function(#4241). So we have to implement window function and hopping. Do you have plan to support STFT?

[1] OORD, Aaron van den, et al. Parallel WaveNet: Fast High-Fidelity Speech Synthesis. arXiv preprint arXiv:1711.10433, 2017.
[2] CHOROWSKI, Jan, et al. On Using Backpropagation for Speech Texture Generation and Voice Conversion. arXiv preprint arXiv:1712.08363, 2017.

**Comment by kmaehashi** [comment]
Thank you for the request. Currently we have no concrete plan to implement STFT, but it is nice to have one in Chainer. We really appreciate for such contributions.
